cmd/finch-daemon/router_utils.go

```diff
@@ -0,0 +1,115 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+
+	"github.com/containerd/containerd"
+	"github.com/containerd/containerd/namespaces"
+	"github.com/containerd/nerdctl/pkg/api/types"
+	"github.com/containerd/nerdctl/pkg/config"
+	toml "github.com/pelletier/go-toml/v2"
+	"github.com/runfinch/finch-daemon/api/router"
+	"github.com/runfinch/finch-daemon/internal/backend"
+	"github.com/runfinch/finch-daemon/internal/service/builder"
+	"github.com/runfinch/finch-daemon/internal/service/container"
+	"github.com/runfinch/finch-daemon/internal/service/exec"
+	"github.com/runfinch/finch-daemon/internal/service/image"
+	"github.com/runfinch/finch-daemon/internal/service/network"
+	"github.com/runfinch/finch-daemon/internal/service/system"
+	"github.com/runfinch/finch-daemon/internal/service/volume"
+	"github.com/runfinch/finch-daemon/pkg/archive"
+	"github.com/runfinch/finch-daemon/pkg/ecc"
+	"github.com/runfinch/finch-daemon/pkg/flog"
+	"github.com/spf13/afero"
+)
+
+// handleNerdctlGlobalOptions gets nerdctl config value from nerdctl.toml file.
+func handleNerdctlGlobalOptions(cfg *config.Config, options *DaemonOptions) error {
```

Comment:
Can we make this more generic in case we add more daemon config options than what nerdctl supports?

Maybe `handleConfigOptions` or similar

Reply:
will rename it.